src/DynamicFormsProvider.php

```diff
@@ -18,6 +18,11 @@ public function register()
             return new ComponentRegistry;
         });
 
+        $this->app->singleton(JSONLogicInitHelper::class, function ($app) {
```

Comment:
If you're not doing any additional config (e.g. passing stuff to the constructor), you do not need to register it with the container -- it's just available implicitly.

Reply:
I'll remove it since its unnecessary but both FileComponentRegistry and ComponentRegistry don't pass things into the constructor and don't need additional config but are still there so I don't fully get the difference

Reply:
Oh, I misread -- didn't realize it was registering a singleton. This is fine.